src/main/java/core/basesyntax/service/impl/CsvReader.java

```diff
@@ -0,0 +1,43 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.strategy.OperationStrategy;
+import core.basesyntax.strategy.StrategyStorage;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+
+public class CsvReader {
```

Comment:
```suggestion
public class FileReaderImpl implements FileReader {
```
цей клас має залежати від абстракції

він повинен також лише читати дані з файла

а окремий сервіс парсити строки в транзакції